src/pyhf/workspace.py

```diff
@@ -286,16 +286,17 @@ class Workspace(_ChannelSummaryMixin, dict):
 
     valid_joins = ['none', 'outer', 'left outer', 'right outer']
 
-    def __init__(self, spec, **config_kwargs):
+    def __init__(self, spec, validate: bool = True, **config_kwargs):
         """Workspaces hold the model, data and measurements."""
```

Comment:
How does the documentation here work? I am not sure where to document the `validate` kwarg. Should I just expand this existing docstring, and also add `spec`?

Reply:
Yeah, the docstrings get a bit fincky for the classes. What you suggest sounds great, and you can take a look at https://scikit-hep.org/pyhf/_generated/pyhf.patchset.PatchSet.html for an example of how we've got things kinda working now.

Reply:
I've updated the docstring, it's also quite similar to the one for the `Model` constructor.